workshop 01 - Introduction to Quantum Computing/README.md

```diff
@@ -5,3 +5,11 @@
 A shallow dive into the world of Quantum Computing! We explore what Quantum can
 achieve, what makes it better than it's classical counterpart? After convincing
 you all, we have made a special hands-on experience session, tailored for you :)
+
+This workshop had talks given by our Contributors and the powerpoint presentations for the same are 
+stored as such 
+
+| Topic | Speakers |
+--|--
+| 1. [Introduction to Quantum Computing](intro-to-quantum-computing.pptx)| [Syed Farhan Ahmed](https://github.com/born-2learn), [Pallavi Agarwal](https://github.com/pallaviagrawal1729), [Madhav Jivrajani](https://github.com/MadhavJivrajani), [Rohit Ashiwal](https://github.com/r1walz)|
```

Comment:
Pallavi and I talked about the QKD and introduction to Q#. IMO, they should enjoy a little bit of space for themselves? Please, maintain the sequence of appearance.

Farhan -> Madhav -> Ricardo -> Pallavi -> Me (IIRC)